Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -16,21 +16,13 @@
       <SubType>Code</SubType>
     </Compile>
     <Compile Include="$(MSBuildThisFileDirectory)Issue3262.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8715.xaml.cs">
```

Comment:
I think the rebase on this one isn't completely correct

Reply:
@PureWeen I think it's correct now. Can you take a look, please?